README.md

```
@@ -1,2 +1,82 @@
-# BE
-🧀 Kusitms 32nd Cheeeese Backend Repository 🧀
+# 🧀 치이이즈  : 딱 7일만 열리는 특별한 공유 앨범 서비스
+> 🔗 서비스 링크: https://say-cheese.me
+
+![img.png](img.png)
+
+## 🧑‍🤝‍🧑 Backend Members
+|**우다현** |                                                     **주정빈**                                                      |
+| :--------: |:----------------------------------------------------------------------------------------------------------------:|
+| <a href="https://github.com/dahyun24"><img src="https://avatars.githubusercontent.com/u/123882512?v=4" width="150"> | <a href="https://github.com/zyovn"><img src="https://avatars.githubusercontent.com/u/166782961?v=4" width="150"> |
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)